kairon/api/app/routers/bot/data.py

```
@@ -549,4 +551,26 @@ async def get_media_ids(
         return Response(message="List of media ids", data=media_ids)
     except Exception as e:
-        raise AppException(f"Error while fetching media ids: {str(e)}")
\ No newline at end of file
+        raise AppException(f"Error while fetching media ids: {str(e)}")
+
+
+@router.delete("/{channel}/media/{media_id}", response_model=Response)
+async def delete_media_data(
+        channel: ChannelTypes,
+        media_id: str,
+        current_user: User = Security(Authentication.get_current_user_and_bot, scopes=DESIGNER_ACCESS)
```

Ruff (0.13.1)

Do not perform function call `Security` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)